### PR TITLE
[MS] Add workspace sharing modal

### LINKED ISSUE #4843: Add workspace sharing modal
https://www.figma.com/file/XW3SNWHV8BhPsPJpsOCV2h/Application?type=design&node-id=21904-32618&mode=design&t=DBTE0VslFyAlRvsb-4

### PR DESCRIPTION
**Pull request by Max-7**
Closes #4843

- [X] Keep changes in the pull request as small as possible
   
- [X] Ensure the commit history is sanitized
   
- [X] Give a meaningful title to your PR
- [X] Link any related issue in the description

### CODE REVIEW

**Review by NicoTuxx — CHANGES_REQUESTED** [review]

File: client/src/locales/fr-FR.json [hunk 1]
```diff
@@ -54,7 +54,7 @@
         "pageTitle": "Créer une organisation",
         "fullname": "Nom complet",
         "fullnamePlaceholder": "Jean MARTIN",
-        "email": "Email",
+        "email": "Courrier",
```
Comment (NicoTuxx) [inline client/src/locales/fr-FR.json:57]:
```suggestion
        "email": "Courriel",
```

File: client/src/theme/variables.css [hunk 2]
```diff
@@ -777,7 +777,6 @@ body {
 ::-webkit-scrollbar {
   width: 1em;
   border-radius: 1em;
-
```
Comment (NicoTuxx) [inline client/src/theme/variables.css]: Why backgrounds were remove here btw?

**Review by NicoTuxx — CHANGES_REQUESTED** [review]

File: client/src/theme/variables.css (hunk 2 above)
Comment (Max-7) [reply client/src/theme/variables.css]: Tests by @fabienSvtr, I reverted the changes.

**Review by NicoTuxx — APPROVED** [review]

File: client/src/locales/fr-FR.json [hunk 3]
```diff
@@ -54,7 +54,7 @@
         "pageTitle": "Créer une organisation",
         "fullname": "Nom complet",
         "fullnamePlaceholder": "Jean MARTIN",
-        "email": "Email",
+        "email": "Courriel",
```
Comment (fabienSvtr) [inline client/src/locales/fr-FR.json:57]: The word "Email" seems very "old", we should use a more current word, "E-mail".

**Review by fabienSvtr — CHANGES_REQUESTED** [review]

**Review by fabienSvtr — APPROVED** [review]

**Pull request merged**